src/core/properties/cameraproperty.cpp

```diff
@@ -53,19 +54,45 @@ CameraProperty::CameraProperty(const std::string& identifier, const std::string&
+                  }())
+    , camera_{factory_->create(cameraType_)}
+    , defaultCamera_{}
     , cameraActions_("actions", "Actions", buttons())
-    , lookFrom_("lookFrom", "Look from", eye, -vec3(100.0f), vec3(100.0f), vec3(0.1f),
-                InvalidationLevel::InvalidOutput, PropertySemantics{"SphericalSpinBox"})
-    , lookTo_("lookTo", "Look to", center, -vec3(100.0f), vec3(100.0f), vec3(0.1f),
-              InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
-    , lookUp_("lookUp", "Look up", lookUp, -vec3(1.f), vec3(1.f), vec3(0.1f),
-              InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
-    , aspectRatio_("aspectRatio", "Aspect Ratio", 1.0f, 0.0f, std::numeric_limits<float>::max(),
-                   0.01f, InvalidationLevel::InvalidOutput, PropertySemantics::Text)
-    , nearPlane_("near", "Near Plane", 0.1f, 0.001f, 10.f, 0.001f)
-    , farPlane_("far", "Far Plane", 100.0f, 1.0f, 1000.0f, 1.0f)
+    , lookFrom_(
+          "lookFrom", "Look from", [this]() { return camera_->getLookFrom(); },
+          [this](const vec3& val) { camera_->setLookFrom(val); },
+          {-vec3(100.0f), ConstraintBehavior::Ignore}, {vec3(100.0f), ConstraintBehavior::Ignore},
+          vec3(0.1f), InvalidationLevel::InvalidOutput, PropertySemantics{"SphericalSpinBox"})
+    , lookTo_(
+          "lookTo", "Look to", [this]() { return camera_->getLookTo(); },
+          [this](const vec3& val) { camera_->setLookTo(val); },
+          {-vec3(100.0f), ConstraintBehavior::Ignore}, {vec3(100.0f), ConstraintBehavior::Ignore},
+          vec3(0.1f), InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
+    , lookUp_(
+          "lookUp", "Look up", [this]() { return camera_->getLookUp(); },
+          [this](const vec3& val) { camera_->setLookUp(val); },
+          {-vec3(1.0f), ConstraintBehavior::Immutable}, {vec3(1.0f), ConstraintBehavior::Immutable},
+          vec3(0.1f), InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
+    , aspectRatio_(
+          "aspectRatio", "Aspect Ratio", [this]() { return camera_->getAspectRatio(); },
+          [this](const float& val) { camera_->setAspectRatio(val); },
+          {0.0f, ConstraintBehavior::Immutable},
+          {std::numeric_limits<float>::max(), ConstraintBehavior::Immutable}, 0.01f,
+          InvalidationLevel::InvalidOutput, PropertySemantics::Text)
+    , nearPlane_(
+          "near", "Near Plane", [this]() { return camera_->getNearPlaneDist(); },
+          [this](const float& val) { camera_->setNearPlaneDist(val); },
+          {0.001f, ConstraintBehavior::Ignore}, {10.0f, ConstraintBehavior::Ignore}, 0.001f)
+    , farPlane_(
+          "far", "Far Plane", [this]() { return camera_->getFarPlaneDist(); },
+          [this](const float& val) { camera_->setFarPlaneDist(val); },
+          {1.0f, ConstraintBehavior::Ignore}, {1000.0f, ConstraintBehavior::Ignore}, 1.0f)
```

Comment:
Increase max bound. Default of it regular value should be 10000...

Reply:
Why this is only for the slider...